### PR TITLE
Signpost the current page as a Convene Space more clearly

### PR DESCRIPTION
**Pull request by zspencer**
See: https://github.com/zinc-collective/convene/issues/160

This builds on @hangtwenty's initial shift; while putting the responsibility into the existing helpers for page-titles; which should allow us to more clearly sign-post which Space & Room a particular Tab is. 

It also drops the Feature into the code-base; so that it's more clear that navigation between external-systems and Convene is something we care about; and may want to more refine with additional intentions.

### CODE REVIEW

**Comment by zspencer** [comment]
Whichever of the @zinc-collective/convene-maintainers merges  this, please take care not to erase @hangtwenty's contribution. I don't quite get how the "Co-authored-by" gets auto-filled on squash merges. I think it pulls each authors email from their commit, but I've noticed it sometimes doesn't work quite the way I expect.

**Review by anaulin — APPROVED** [review]

**Comment by anaulin** [comment]
I wasn't sure if during "Squash and merge" I was going to get the option to add co-authorship info, so out of an abundance of caution I made a "manual squash" to end up with one commit with the correct co-authorship info. But it looks like that also added me as a co-author? Which is not really what I intended. Sigh. Sorry.

I'm going to try to merge this now, and see if I can add the co-authorship info during the squash+merge step.

**Comment by anaulin** [comment]
For anyone following along at home: "Squash and merge" did give me the ability to add or change the co-authorship information, and it had correctly already filled it in with what was in my commit (and it did not add me to the authors 😌 ). Here is the resulting merge commit: https://github.com/zinc-collective/convene/commit/8411ab240175f7bd951fabf7eb3a1cb6cf3f27e7

**Comment by zspencer** [comment]
Thank you so much @anaulin!

**Pull request merged**
